code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,99 @@
+import request from 'supertest';
+import express from 'express';
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+import models from '../../setup/models'
+import bcrypt from 'bcrypt';
+import db from '../../setup/database';
+
+describe("user mutations", () => {
+  let server = express();
+
+  beforeAll(async() => {
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false
+      })
+    );
+    await models.User.destroy({ where: {}})
+  });
+
+  beforeEach(async() => {
```

Comment:
The setup and teardown looks great here

Reply:
Yeah, agreed. Good work getting that in place.